common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -739,18 +740,17 @@ describe 'Problem', ->
         deferred.promise()
 
       runs.call(this).then(->
```

Comment:
Is this bit of code (lines 742-744) doing anything now? I'm having trouble understanding the empty "then".